velox/functions/prestosql/tests/ArrayMinTest.cpp

```diff
@@ -36,11 +36,21 @@ class ArrayMinTest : public FunctionBaseTest {
   }
 
   void testDocExample() {
```

Comment:
nit: is there used anywhere else? if not we can just move this into `TEST_F(ArrayMinTest, docs)`